_data/people.yml

```diff
@@ -174,3 +174,21 @@
   github: drounce
   url: https://www.cmu.edu/cee/people/faculty/rounce.html
   visible: true
+
+- first_name: Li
+  last_name: Fei
+  institution: Institute of Tibetan Plateau Research, Chinese Academy of Sciences.
+  github: Keeptg
+  visible: true
+
+- first_name: Sarah
+  last_name: Hanus
+  institution: University of Zurich
+  github: sarah-hanus
+  visible: true
+
+- first_name: Mat
+  last_name: Cast
```

Comment:
```suggestion
  last_name: Castellani
```

Reply:
ah you did not send full name 🤔 ... will modify this and commit

Reply:
see changes in the commit below